src/backend/aspen/app/views/tests/test_samples_view.py

```diff
@@ -39,7 +42,107 @@ def test_samples_view(
                 "private_identifier": sample.private_identifier,
                 "public_identifier": sample.public_identifier,
                 "upload_date": api_utils.format_datetime(sequencing_read.upload_date),
-                "gisaid": sequencing_read.accessions()[0].public_identifier,
+                "gisaid": {
+                    "status": "accepted",
+                    "gisaid_id": sequencing_read.accessions()[0].public_identifier,
+                },
+            }
+        ]
+    }
+    assert expected == json.loads(res.get_data(as_text=True))
+
+
+def test_samples_view_gisaid_rejected(
+    session,
+    app,
+    client,
+):
+    group = group_factory()
+    user = user_factory(group)
+    sample = sample_factory(group)
+    # Test no GISAID accession logic
+    sequencing_read = sequencing_read_factory(sample, accessions={})
```

Comment:
doesn't this imply "not_submitted" and not "rejected"?

Reply:
Well this hits the final case in the _format_gisaid_accession, where there was no accession, or consuming workflow within a 4 day period. But this comment might imply theres another case of "eligible but not yet submitted"?